test/distributed/pipelining/test_schedule_multiproc.py

```diff
@@ -821,3 +822,4 @@ def dw_runner():
             nprocs=world_size,
             args=(world_size, rdvz_file),
         )
+    run_tests()
```

Comment:
I'm not sure this should be here or in the other distributed files, cc @wconstab

Reply:
My only goal here was consistency. I make 127 instances of run_tests() in test/distributed but happy to erase them. Or would it just be the pipelining subdir where there are 6?. @wconstab